python/pyspark/worker.py

```diff
@@ -604,7 +604,7 @@ def process():
         # reuse.
         TaskContext._setTaskContext(None)
         BarrierTaskContext._setTaskContext(None)
-    except Exception:
+    except BaseException:
```

Comment:
Can you change `except Exception:` at 621 line to `except BaseException` too?

Reply:
at 621 line seems no need to cache BaseException, because daemon.py will cache SystemExit at 75 line

Reply:
updated